### DISCUSSION TITLE
Sync files via CLI

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3044] [Feature Requests]
Would be really nice if you could sync files via CLI, which can be really slow in CP.  Two separate situations:

1. Typical directory sync, especially after you've added new image sizes and such.
2. After upgrade sync where you want to turn compatibility mode off and have a ton of files that need to be changed.  This can take forever with big directories.

**Comment by intoeetive** [comment]
`2` is already imlemented in `sync:file-usage` command
`1` is getting added in this PR: https://github.com/ExpressionEngine/ExpressionEngine/pull/3615
